portfolio/src/main/webapp/index.html

```diff
@@ -82,6 +82,8 @@ <h1><strong>Josiah Paintsil's World</strong></h1>
             </p>
           </article>
         </div>
+
+        <div><a href= "/strservlet"><button>Run Servlet</button></a></div>
```

Comment:
["Use a new line for every block, list, or table element, and indent every such child element."](https://google.github.io/styleguide/htmlcssguide.html#General_Formatting)

```suggestion
        <div>
          <a href= "/strservlet"><button>Run Servlet</button></a>
        </div>
```